src/apps/backend/modules/account/internal/account-util.ts

```diff
@@ -25,4 +26,12 @@ export default class AccountUtil {
   ): Promise<boolean> {
     return bcrypt.compare(password, hashedPassword);
   }
+
+  public static getContactNumberString(
+    contactNumber: ContactNumber,
+  ): string {
+    const { countryCode, phoneNumber } = contactNumber;
+
+    return `${countryCode} ${phoneNumber}`;
+  }
```

Comment:
This is not the right place for this.  Expose `toString` method on `ContactNumber` (should be renamed as `PhoneNumber`) and have that return `${countryCode} ${phoneNumber}`